.eslintrc.js

```diff
@@ -2,8 +2,18 @@
 
 const config = {
   root: true,
-  env: { commonjs: true, node: true, browser: true },
-  extends: ['ackama', 'ackama/react'],
+  env: {
+    commonjs: true,
+    node: true,
+    browser: true
+  },
```

Comment:
```suggestion
  env: { commonjs: true, node: true, browser: true },
```

🐘 might as well restore the original layout to reduce the diff noise